traffic-control-openshift/README.md

```diff
@@ -0,0 +1,62 @@
+# Traffic Control on OpenShift
+
+## What do you need to start?
+- Get a OpenShift ready VM [here](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo. You need to __reboot__ the VM after you build it.
```

Comment:
Avoid ["here" links](https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/#here-conceals-what-users-are-clicking)